### PR TITLE
Create Random function

### PR DESCRIPTION
**Pull request by alper**
As discussed here: https://github.com/SeaQL/sea-orm/discussions/1009#discussioncomment-3523213

### CODE REVIEW

**Review by ikrivosheev — APPROVED** [review]
@alper hello! Thank you for contribution! LGTM!

File: src/backend/mysql/query.rs [hunk 1]
```diff
@@ -95,6 +95,10 @@ impl QueryBuilder for MysqlQueryBuilder {
         "()"
     }
 
+    fn random_function(&self) -> &str {
+        "RAND()"
```
Comment (ikrivosheev) [inline src/backend/mysql/query.rs:99]: I don`t think brackets are needed here.

Comment (alper) [reply src/backend/mysql/query.rs:99]: I think they are! 

```
foursquaremaps=# select RANDOM;
ERROR:  column "random" does not exist
LINE 1: select RANDOM;
```

The parens need to be there and they're not added. The other example does work without parens:

```
foursquaremaps=# select CURRENT_TIMESTAMP;
       current_timestamp       
-------------------------------
 2022-09-01 16:38:11.467524+02
(1 row)
```

Comment (alper) [reply src/backend/mysql/query.rs:99]: Query generation does not insert the paren automatically and seems roundabout if it's a function with zero arguments.

Comment (alper) [reply src/backend/mysql/query.rs:99]: There's also `"(DEFAULT)"` where the parens are included in the output immediately.

Comment (ikrivosheev) [reply src/backend/mysql/query.rs:99]: Ok, thank you for explain!

**Comment by alper** [comment]
OK. Who'll merge it?

**Comment by ikrivosheev** [comment]
> OK. Who'll merge it?

@billy1624 @tyt2y3 can merge. We are waiting for them)

**Review by billy1624 — APPROVED** [review]
Thanks!! @alper :)

File: src/backend/mysql/query.rs (hunk 1 above)
Comment (tyt2y3) [reply src/backend/mysql/query.rs:99]: Sadly, I think it's only because of `CURRENT_TIMESTAMP` which arguably is not a function

**Pull request merged**